src/openshift/component.ts

```diff
@@ -61,13 +61,20 @@ export class Component extends OpenShiftItem {
     }
```

Comment:
I would change only command, keep the messages the same.

Reply:
The message is added so that the user is notified what exactly delete action will do. We already have a command for undeploy too. So to give `Delete` action more clarity, I have added the message specific to `pushed` components.

Reply:
I'm gravitating towards the current messages, feels to me like the distinction between undeploy and delete is quite intuitive. ¯\_(ツ)_/¯

Reply:
Sure thing then, will then roll back those changes.